scripts/contributors.sh

```
@@ -0,0 +1,272 @@
+"
+
+  for author in "${!AUTHOR_ORDER[@]}"; do
+    entries="${AUTHOR_PRS[$author]}"
+    # Count entries for this author
+    entry_count=$(echo "$entries" | wc -l | tr -d ' ')
+
+    if [[ "$entry_count" -eq 1 ]]; then
+      # Single PR: use the PR title as description
+      desc=$(echo "$entries" | head -1)
+      # Extract just the title part (before the PR reference)
+      title_only=$(echo "$desc" | sed 's/ (#[0-9]*)$//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)